### ISSUE TITLE
Some items cannot be fully disabled

### ISSUE THREAD

**Issue by KeepOnDigging** [OpenMods/OpenBlocks#358]
Hey, great mod. There are some fantastic features here. I especially love the simplicity of the Tanks.

For a new world I am creating, I really only wanted a couple of features from this mod, starting with Tanks. When I disabled all the other features, I think I have found some bugs.

I have stripped my installed mods down to just NEI/chickenCore and OpenBlocks.

I have set all block IDs (except blockTankId) to 0.
I have set all item IDs (except itemInfoBookId) to 0.
I set the IDs of the enchantments to 0.
I set B:enableChestLoot=false and I:radioVillagerId=-1 and B:weAreSeriousPeople=false

Now I see that the entries for disabled items have been removed from the starter book. Nice touch!
But I would expect this Unprepared Stencil page to be hidden too:

![](http://s27.postimg.org/76h2hm16r/Screen_shot_2014_03_18_at_9_46_24_PM.png)

And also pages for "The infamous B key" and the three Enchantments still show in the book, even though I disabled the tasty clay, seriousPeople and the enchantments.

In the OpenBlocks creative inventory tab, I still see the Pointer:

![](http://s28.postimg.org/tguvayix9/Screen_shot_2014_03_18_at_9_45_57_PM.png)

I hit R to view the recipes for the Pointer:

![](http://s4.postimg.org/u7x15txfh/Screen_shot_2014_03_18_at_9_45_39_PM.png)
![](http://s8.postimg.org/jbkvfu0z9/Screen_shot_2014_03_18_at_9_45_47_PM.png)

In survival, this effectively disables the iron shovel because a Pointer is crafted from the ingot and two sticks.

I guess these log entries are related to this last issue:

```
2014-03-18 21:57:44 [INFO] [STDOUT] CONFLICT @ 0 item slot already occupied by net.minecraft.item.ItemSpade@325b954d while adding openblocks.common.item.ItemOBGeneric@1aa70c0f
2014-03-18 21:57:44 [INFO] [fml.ItemTracker] The mod OpenBlocks is overwriting existing item at 256 (net.minecraft.item.ItemSpade from Minecraft) with openblocks.common.item.ItemOBGeneric
2014-03-18 21:57:44 [INFO] [STDOUT] CONFLICT @ 0 item slot already occupied by openblocks.common.item.ItemOBGeneric@1aa70c0f while adding openblocks.common.item.ItemOBGenericUnstackable@178f9f80
2014-03-18 21:57:44 [INFO] [fml.ItemTracker] The mod OpenBlocks is overwriting existing item at 256 (openblocks.common.item.ItemOBGeneric from OpenBlocks) with openblocks.common.item.ItemOBGenericUnstackable
```

But if I don't change the IDs of these two items to 0, I see lots of OpenBlocks items in the creative tab… glider wings, crane stuff, pencil etc.

Also interesting to see all these interfaces being added to disabled items, although I guess this is functionally irrelevant:

```
2014-03-18 21:57:44 [INFO] [OpenModsCore] Adding interface net/minecraftforge/fluids/IFluidHandler to openblocks/common/tileentity/TileEntityTank
2014-03-18 21:57:45 [INFO] [OpenModsCore] Adding interface net/minecraftforge/fluids/IFluidHandler to openblocks/common/tileentity/TileEntitySprinkler
2014-03-18 21:57:45 [INFO] [OpenModsCore] Adding interface net/minecraft/inventory/IInventory to openblocks/common/tileentity/TileEntitySprinkler
2014-03-18 21:57:45 [INFO] [OpenModsCore] Adding interface net/minecraft/inventory/ISidedInventory to openblocks/common/tileentity/TileEntityVacuumHopper
2014-03-18 21:57:45 [INFO] [OpenModsCore] Adding interface net/minecraftforge/fluids/IFluidHandler to openblocks/common/tileentity/TileEntityVacuumHopper
etc for about 20 lines
```

Thanks for the great mod :)

KeepOnDigging!

**Comment by khionu** [comment]
I feel this issue should be reopened. Even if a single one of these cannot be disabled fully, it's a bug. My current situation is that I prefer the OpenGlider's implementation of Gliders, but the recipe conflicts with the Wings. Even using an NBT editor, I can't find enough information to be able to disable the recipe with modtweaker.

**Comment by boq** [comment]
Since 1.12 every item can be disabled. And every recipe can be modified with resourcepack. Anything below 1.12 is no longer supported, so this bug remains closed.

**Comment by khionu** [comment]
I am using the latest available on CurseForge for 1.12, and disabling the Glider does not disable the Glider Wings. I assumed this was the same issue because the ID for the Glider Wings I craft is "openblocks:generic".
